docs/release-notes/self-hosted/self-hosted-v7.0.0.md

```diff
@@ -0,0 +1,122 @@
+
+    ![Refreshing the list of repositories on Codacy](../images/cy-5876.png)
+
+## Product enhancements
+
+-   GitLab project members with the Maintainer role can now perform the following operations on Codacy:
+
+    -   Adding the project
+    -   Configuring the project
+    -   Changing the following analysis settings: ignoring issues and files, configuring code patterns, configuring file extensions, and managing branches
+
+    [See the updated permissions](https://docs.codacy.com/v7.0/organizations/roles-and-permissions-for-synced-organizations/) for all GitLab roles. (CY-5876)
+
+-   Included ESLint 8 as a new supported tool and deprecated ESLint 7. For now, all repositories will continue to use ESLint 7 by default. (CY-5848)
+
+-   The [**Repositories list** page](https://docs.codacy.com/v7.0/organizations/managing-repositories/) now displays a warning icon to improve the visibility of warnings and errors affecting the repositories. (CY-5797)
+
+    ![Repositories with warnings on the Repositories list page](../images/cy-5797.png)
+
+-   The [**Open pull requests** area](https://docs.codacy.com/v7.0/repositories/repository-dashboard/#open-pull-requests) on the Repository Dashboard now displays more details about the open pull requests. (CY-5796)
+
+    ![Open pull requests area on the Repository Dashboard](../images/cy-5796.png)
+
+-   The [**Access management** page](https://docs.codacy.com/v7.0/getting-started/which-permissions-does-codacy-need-from-my-account/#revoking-access-to-integrations) under your account settings now also allows adding new Git providers. (CY-5795)
+
+-   Improved the loading times of the Organization Overview and Repositories list pages using a short-lived cache in the user's browser. (CY-5793)
+
+## Bug fixes
+
+-   Fixed an issue that caused inconsistencies on the last updated date when listing GitHub repositories. Now, the last updated date is the [date of the last push to the repositories](https://docs.codacy.com/v7.0/organizations/organization-overview/#last-updated-repositories). (CY-5784)
+-   dartanalyzer now supports including the packages [lints](https://pub.dev/packages/lints){: target="_blank"} and [flutter_lints](https://pub.dev/packages/flutter_lints){: target="_blank"} in the `analysis_option.yaml` configuration file. (CY-5626)
+-   The re-analyze button is now hidden on repositories that are running analysis through a build server. (CY-4205)
+
+## Tool versions
+
+This version of Codacy Self-hosted includes the tool versions below. The tools that were updated on this version are highlighted in bold:
+
+-   Ameba 0.13.1
+-   Bandit 1.7.0
+-   Brakeman 4.3.1
+-   bundler-audit 0.6.1
+-   Checkov 2.0.399
+-   Checkstyle 8.44
+-   Clang-Tidy 10.0.1
+-   CodeNarc 2.2.0
+-   CoffeeLint 2.1.0
+-   Cppcheck 2.2
+-   Credo 1.4.0
+-   CSSLint 1.0.5
+-   **dartanalyzer 2.16.1 (updated from 2.15.1)**
+-   detekt 1.19.0
+-   **ESLint 8.10.0 (new)**
+-   ESLint (deprecated) 7.32.0
```

Comment:
Should we mark the depracted as bold and red :D?

Reply:
I would consider adding the "Will be removed on a future release" as well here. Also, I don't mind the red and bold at all

Reply:
Here I'm concerned that we don't have a set date for dropping the ESLint 7 version (nor a defined plan to help customers migrate to ESLint 8).

So, I would only call extra attention to this if there is a clear course of action that customers can take at the time they upgrade to Codacy Self-hosted 7.0.0. :thinking: